shoop/core/fields/__init__.py

```diff
@@ -52,12 +53,17 @@ def deconstruct(self):
         return (name, path, args, kwargs)
 
 
-class MoneyField(models.DecimalField):
+class CurrencyField(models.CharField):
```

Comment:
I may need some help with that. For now I just wanted the CurrencyFields to be separated from general purpose CharFields.